### ISSUE TITLE
Add service web page

### ISSUE THREAD

**Issue by TwinProduction** [TwinProduction/gatus#78]
Displaying a list of all services is great for an overview, but with the addition of the uptime over the past 7 days, 24 hours and 1 hour, a page for displaying a single service could be useful.

This could be added with just pure HTML/CSS/JS, _**or**_ perhaps with a framework like React or Angular.

# Requirements
The page must have the following data.

## The service status checks 
Same as on the list of services

## The service's uptime percentage
Over the past 7 days, 24 hours and 1 hour.

## A link to go back to the list of the services
... because it makes sense

## A section with the last 10 events
This is what it should look like:
```
<up.png> The service is back up
<start timestamp>

<down.png>The service went down for 10 minutes
<start timestamp>
```

## Extra
It would be nice to have a graph of the response time.

**Comment by TwinProduction** [comment]
So far so good
![image](https://user-images.githubusercontent.com/15699766/106074015-1e73b000-60d9-11eb-8b34-8a9e2815a208.png)

**Comment by TwinProduction** [comment]
Released in [v2.0.0](https://github.com/TwinProduction/gatus/releases/tag/v2.0.0)
